infrastructure-playbooks/lv-create.yml

```diff
@@ -38,6 +38,8 @@
     package:
       name: lvm2
       state: present
+    register: result
+    until: result is succeeded
```

Comment:
This might lead to infinite loop. It would be nice to add `retry` and possibly even `delay`. See [this page](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#do-until-loops) in docs.

Reply:
Nop, look closely at the link you sent: `The default value for “retries” is 3 and “delay” is 5.`